project/src/data/technicalArea.js

```diff
@@ -0,0 +1,218 @@
+const getTechnicalAreaData = function() {
+    return [
+        {
+            items: [  
+                {
+                  label: 'Classes',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Exception Handling',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Version Control',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Access Modifiers',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Design patterns',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Issue Tracking',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Polymorphism',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'RegEx',
+                  options: ["0","1","2","3"]
+            }],
+            legends: 'OOP, Design Patterns'
+        },
+    
+        {
+            items: [
+                {
+                  label: 'Protocol',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Response Codes',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Rest',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Headers',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Request Methods',
+                  options: ["0","1","2","3"]
+                },
+                {
+                  label: 'Sessions & Cookies',
```

Comment:
Just wondering if these values get the HTML escaping treatment before being shoved into the DOM. Otherwise, they might break the whole page (e.g., what happens if any of these labels contains the text `<script>`?).

Reply:
they won't, I'll have to change the & into &amp; and I did that in html, I have no explanation why I changed it :))